INSTALL.md

```diff
@@ -52,22 +69,51 @@ To skip the secrets taks, just pass:
 
 The secret keys that have to be defined are:
 
-* backend_url
-* base_url
-* backend_token
-* secret_key
-* info_email
-* ssl_stapling_resolver
-* google_analytics
-
+* backend_url: The URL for requesting the backend. Ex: http://kernelci-backend/
+* base_url: The URL for requesting the frontend. Ex: http://kernelci-frontend/
+* backend_token: A GET+POST token. (See below for generating them)
+* secret_key:  should be set to a random string, it is used internally by Flask.
+* master_key: The password used for generating tokens before any admin token was created.
+* info_email:
+* ssl_stapling_resolver:
+* google_analytics:
 
```

Comment:
For these last three, they are shown as empty.  Does that mean they're optional?